src/assets/scss/_multiLineEllipsis.scss

```diff
@@ -0,0 +1,32 @@
+// Taken from http://hackingui.com/front-end/a-pure-css-solution-for-multiline-text-truncation/
+
+/* mixin for multiline */
+@mixin multiLineEllipsis($lineHeight: 1.2em, $lineCount: 1, $bgColor: white){
+  overflow: hidden;
+  position: relative;
+  line-height: $lineHeight;
+  max-height: $lineHeight * $lineCount;
+  text-align: justify;
```

Comment:
Definitely need to avoid the justified text. That never looks good on the web, too many rivers.

Reply:
The CSS doesn't know where to add an ellipsis otherwise, hence the bottom right.